### ISSUE TITLE
Formatting cell in search for multi-value

### ISSUE THREAD

**Issue by hartois** [glpi-project/glpi#1166]
Hello. 
I made some changes (needed for my users) in the display format of multi-values in search results. 
Perhaps it will be interesting.
Tooltip appears on click.
![screen0](https://cloud.githubusercontent.com/assets/11641656/19411091/3ad45a2c-9303-11e6-816f-65f3e01cf1d2.png)

Commit: https://github.com/hartois/glpi/commit/ea4dca4d20fec5c9018b6dbc4385e5d4fcd1e421

**Comment by flegastelois** [comment]
Hi, 
Very interesting, I agree with you, can you make a Pulk Request ?
Cheers,

**Comment by hartois** [comment]
I can make a PR, but I realized that I made mistakes. I use the reverse sort of values. But it can be used for comments only. How can I check the type of the field?

**Comment by hartois** [comment]
Additional changes: collapse cell to two lines and add row-expander, change tooltip icon.

Collapsed row:
![screen1](https://cloud.githubusercontent.com/assets/11641656/19415987/5ca277f0-938b-11e6-99f1-d9dd9d64d6cf.png)

Expanded row:
![screen2](https://cloud.githubusercontent.com/assets/11641656/19415989/5ca35576-938b-11e6-9c29-6b2f2537df98.png)

Show list tooltip:
![screen3](https://cloud.githubusercontent.com/assets/11641656/19415988/5ca3526a-938b-11e6-8032-9a229466ded6.png)

**Comment by hartois** [comment]
Can I still make PR for this changes without ordering values?

**Comment by flegastelois** [comment]
@orthagh what is your opinion on this feature?

**Comment by hartois** [comment]
@orthagh please tell your opinion. If it is not interested - I do not raise this issue. If interested - I can make PR. I do not want to clutter up the unnecessary PR.

**Comment by orthagh** [comment]
Ok for me. I approve this change for the master branch.
Push a pr linked to this issue and we can continue discussion on it.

**Comment by hartois** [comment]
I have found some problems with expanders for fields that there is no need to collapse - eg Date.
Which fields would be better to render with "white-space: nowrap"...
I think - is to abandon the idea of collapse/exspand cells and make only the pop-up window for the cells with more than one meaning.
It remains an open question of sorting values for certain fields. At this point I decided this by new case "glpi_ticketfollowups.content" in the Search :: getItem()
